site/pages/sysadmin/manage-organisations.page.tsx

```diff
@@ -79,11 +79,13 @@ const ManageOrganisations = ({ orgList, csrfToken }: ManageOrganisationsProps):
                     ]}
                 />
             ) : null}
-            <Table
-                caption={{ text: "Manage organisations", size: "m" }}
```

Comment:
Looking at the designs I think this should be "govuk-body", the designs do have a lower font weight but I'd ignore that for now

Reply:
Are you referring to the heading "Manage organisations"? i have updated it "govuk-heading-l" as this is used in most of the pages for a heading name.

Reply:
Sorry yeah, commented in the wrong place